charts/argo/templates/workflow-controller-config-map.yaml

```diff
@@ -50,6 +50,9 @@ data:
         {{- if .Values.artifactRepository.s3.useSDKCreds }}
         useSDKCreds: {{ .Values.artifactRepository.s3.useSDKCreds }}
         {{- end }}
+      gcs:
```

Comment:
Is having both the s3 and gcs sections present going to cause issues? do we want to look at conditionally setting either the s3 or gcs configuration depending on what the user is using?